### PR TITLE
Changed AuthPolicy.create_for_modification to raise NotImplmentedError

### PR DESCRIPTION
**Pull request by ErnestoLoma**

### CODE REVIEW

**Review by 0xquark — COMMENTED** [review]
I don't think the files under `docs/` are meant to be committed

**Comment by ErnestoLoma** [comment]
> I don't think the files under `docs/` are meant to be committed

There are already files in docs. @cmgrote ?

File: pyatlan/model/enums.py [hunk 1]
```diff
@@ -2060,3 +2060,12 @@ class SourceCostUnitType(str, Enum):
     CREDITS = "Credits"
     BYTES = "bytes"
     SLOT_MS = "slot-ms"
+
+
+class jsdk_DataQualityType_RVUuI(str, Enum):
```
Comment (cmgrote) [inline pyatlan/model/enums.py:2065]: This is a generated enum from integration tests — should revert to the last version of `enums.py` without this class.

**Review by cmgrote — CHANGES_REQUESTED** [review]
`docs` should stay — that's to generate a separate JavaDocs-equivalent site that just gives info about the entire Python SDK (its classes, operations, etc).

File: pyatlan/model/enums.py (hunk 1 above)
Comment (ErnestoLoma) [reply pyatlan/model/enums.py]: done

**Review by cmgrote — APPROVED** [review]

**Pull request merged**